catroid/src/main/java/org/catrobat/catroid/utils/UtilZip.java

```diff
@@ -173,4 +180,52 @@ public static void deleteZipFile(String fileName, String destinationFolder) {
 			fileToBeDeleted.delete();
 		}
 	}
+
+	public static void unzipTemplate(String projectName, String templateName, String zipFileName) {
+		Log.d(TAG, "zip file name:" + zipFileName);
+		Archiver archiver = ArchiverFactory.createArchiver("zip");
+		File unpackedDirectory = new File(Constants.DEFAULT_ROOT + "/" + templateName);
+		try {
+			archiver.extract(new File(zipFileName), unpackedDirectory);
+		} catch (IOException e) {
+			Log.d(TAG, "Can't extract program", e);
```

Comment:
why is this error not handled anywhere? Will the app go on as if nothing happened?